### PR TITLE
READme.md

### PR DESCRIPTION
**Pull request by Dorcastunmise**
READme.md Edit 
Includes addition of new sections & linkage to CONTRIBUTION.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,3 +77,90 @@ $ npm run dev
...
 
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+
+## Prerequisites 🛠️
+
+Before you begin, ensure you have met the following requirements:
+
+Make sure you have Node.js and npm installed.
+
+  1.  Install Vite tool first
```
Comment (jnadroj) [inline README.md:87]: can we eliminate the part of creating the vite project from scratch?

File: README.md [hunk 2]
```diff
@@ -73,3 +77,90 @@ $ npm run dev
...
+
+  3.  Run:
+  <pre> cd responsive-webapp </pre>
+  <pre> npm install </pre>
+  <pre> npm run dev </pre>
+  Application runs from localhost:http://127.0.0.1:5173/.
+
+  4. Copy and paste content of code file and folders into the project folder
+
+  5. Install devDependencies listed below 👇🏻👇🏼👇🏽👇🏾👇🏿
```
Comment (jnadroj) [inline README.md:100]: can we eliminate the part of creating the vite project from scratch? 
```suggestion

```

File: README.md [hunk 3]
```diff
@@ -73,3 +77,90 @@ $ npm run dev
...
+
+     ## OR
+
+Clone or Download the repository 
+    <pre>git clone https://thxmx-ui.netlify.app/ </pre>
+
```
Comment (jnadroj) [inline README.md:106]:
```suggestion
Clone or Download the repository 
    <pre>git clone https://thxmx-ui.netlify.app/ </pre>

Install dependencies 
        <pre>npm install</pre>
        
Run the proyect 
    <pre>npm run dev</pre>
```

File: README.md [hunk 4]
```diff
@@ -73,3 +77,90 @@ $ npm run dev
...
+2. Packages / devDependencies installation: 
+<pre>npm i vite vite-plugin-dts @vitejs/plugin-react </pre>
+<pre>npm i @babel/core @emotion/css @emotion/react @emotion/styled </pre>  
+<pre>npm i @storybook/addon-a11y @storybook/addon-actions @storybook/addon-docs</pre>
+<pre>npm i @storybook/addon-essentials @storybook/addon-interactions</pre>    
+<pre>npm i @storybook/addon-links @storybook/react @types/no-scroll </pre>
+<pre>npm i @types/node @types/react @types/react-dom</pre> 
+<pre>npm i babel-loader json no-scroll react-focus-lock</pre>  
+<pre>npm i acorn-jsx acorn-jsx </pre>
```
Comment (jnadroj) [inline README.md:119]:
```suggestion
## Run the storybook 

    <pre>npm run storybook</pre>
```

File: README.md [hunk 5]
```diff
@@ -73,3 +77,90 @@ $ npm run dev
...
+TypeScript javascript Library ReactJS Components Microsoft compiler language UI React-ui framework
+frontend Design library-ui
+
+## Contact 👋 
+
+If you want to contact me you can reach me at :
+     Websites
+jordanjaramillo.co (Personal)
+medium.com/@jordanrjdev (Blog)
+dev.to/jordandev (Blog)
```
Comment (jnadroj) [inline README.md:163]:
```suggestion
```

**Review by jnadroj — CHANGES_REQUESTED** [review]
please the comments I posted can be improved by taking your personal considerations into account.

**Review by djra26111990 — APPROVED** [review]

**Pull request merged**